scss/_orange-navbar.scss

```diff
@@ -0,0 +1,62 @@
+// Contents
+//
+// Supra bar
+
+
+// Supra bar
+//
+// Brand exclusive thin navbar that can be used with orange navbar
+.supra {
+  display: none;
+  padding-top: 0;
+  padding-bottom: 0;
+  border-bottom: $navbar-orange-border-bottom solid $gray-700;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+    .container {
+      padding-right: $navbar-orange-padding-x;
+      padding-left: $navbar-orange-padding-x;
+    }
+  }
+
+  .navbar-nav .nav-link {
+    padding: #{map-get($spacers, 2) + $navbar-supra-padding-y} $navbar-supra-padding-x $navbar-orange-link-padding-bottom;
+    font-size: $font-size-sm;
+    line-height: 1;
+
+    &.active::before {
+      right: $navbar-supra-padding-x;
+      bottom: -$navbar-orange-border-bottom;
+      left: $navbar-supra-padding-x;
+    }
+
+    &.nav-icon {
+      padding: divide($spacer, 2) $navbar-orange-icon-padding-x $navbar-orange-icon-padding-bottom;
+
+      svg,
+      img {
+        width: $navbar-supra-icon-size;
+        height: $navbar-supra-icon-size;
+      }
+    }
+
+    .badge {
+      @extend .position-absolute;
+      @extend .top-0;
+      @extend .bg-info;
+      @extend .text-white;
+      @extend .translate-middle;
+      @extend .rounded-pill;
+      padding: $navbar-orange-badge-padding-y $navbar-orange-badge-padding-x;
+      margin-top: $navbar-supra-padding-y;
```

Comment:
padding variable used for margin?

Reply:
Made a duplicate called navbar-orange-supra-padding-y